examples/dht_server.rs

```diff
@@ -75,7 +75,7 @@ fn main() {
 
     let lan_discovery_sender = LanDiscoverySender::new(tx.clone(), server_pk, local_addr.is_ipv6());
 
-    let mut server = Server::new(tx, server_pk, server_sk);
+    let mut server = Server::new(tx, server_pk, server_sk, local_addr.is_ipv6());
```

Comment:
`server.set_ipv6_mode` looks better.

Reply:
set_ipv6_mode() should set Server, DhtFriend, Bucket, Kbucket. This increases complexity of source code.
And there is no benefit for dynamic setting for ip_v6, I think.
Does it need to set ip_v6 dynamically?

Reply:
It's not necessary that `Bucket` contains `is_ipv6_enabled`. If you don't add to `Bucket` it then `set_ipv6_mode` will have to update only `Server` and `Kbucket` (and perhaps we will be able to remove it from `Kbucket` later as well).